server/v2/store/flags.go

```diff
@@ -0,0 +1,8 @@
+package store
+
+const (
```

Comment:
All those were actually in the `app.toml`. Shall we have the module component have its own config then? The command can then read it if it isn't specified.